style.css

```diff
@@ -45,13 +44,104 @@ header {
   text-decoration: none;
 }
 
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-end;
+  background: #6070ff;
+  cursor: pointer;
+  visibility: hidden;
+  opacity: 0;
+  transition: all 0.35s ease-in;
+}
```

Comment:
- [ ] As the previous code reviewer required we need to match the Figma design, I thought because we are struggling a little bit I wanted to explain what I did to solve it, In order to make a background transparent we use `opacity` fortunately in CSS, you can specify opacity inside `rgb` function for example 
```CSS 
background-color: rgb(96 112 255 / 60%)
```
and to give blur effect in the background we can use `backdrop-filter` property
```CSS
backdrop-filter: blur(value_of_px);
```
https://www.loom.com/share/b43fc5df639146beb8f53cfb8796ab60

Reply:
Thank you. I have updated it

Reply:
@Mikelobi I hope it was helpful. If you didn't understand a part please let me know